### ISSUE TITLE
LDM Onboarding

### ISSUE THREAD

**Issue by patrickm02L** [cityofaustin/atd-data-tech#12648]
## Description

Work with the Devs to get spun up locally and document the steps you're taking.

**Comment by patrickm02L** [comment]
@frank please add thoughts about getting this set up

**Comment by frankhereford** [comment]
Thanks @patrickm02L for the reminder on Monday that this was in my court - i had lost track of it in the 'in-scoping' state. 

I'm bringing it into in-progress because we have a meeting on the books for this Friday. Thanks again!

**Comment by frankhereford** [comment]
@roseeichelmann @mddilley and I had an exceptional meeting for almost two hours today. We walked through the LDM from the very beginning, detailing how each step builds on the last and why decisions were made the way they were and answering questions - all that.

It was a lot of information all very quickly, and I made the commitment in the meeting, and I'll make it again - I'm available to the absolute maximum to answer questions and support their work as they start to work on this system. All questions welcome!

I'm so encouraged - just in this initial meeting, ideas came up that will be innovative ways we can leverage the new capabilities of the LDM and how we can improve choices that I've made so far.

@amenity @patrickm02L @chiaberry Thank you to everyone who helped make this new organizational approach to this project a reality - I'm just so excited.
